### PR TITLE
Feature/mauro/stream

### PR DESCRIPTION
**Pull request by MauroHorie**

### CODE REVIEW

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ActivityObjectType.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package io.moj.java.sdk.model.stream;
+
+/**
+ * Enumeration for the activity type.
+ * Created by Mauro on 2016-06-24.
+ */
+public enum ActivityObjectType {
+    LEAVE("Leave"),
```
Comment (skidson) [inline mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ActivityObjectType.java]: I think we need to annotate these with @SerializedName("Leave") for GSON to parse them correctly. It looks like you've kept the Type fields below as a String in the models though and I'm guessing you're opting for the clients to extract them with an ActivityObjectType.fromKey()? I'm OK with this just want to verify though as it is different than how we parse the other enums in the SDK.

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/BaseActivityObject.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    public String getId() {
+        return Id;
+    }
+
+    public void setId(String id) {
+        Id = id;
+    }
+
+    public String getContext() {
```
Comment (skidson) [inline mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/BaseActivityObject.java]: This is going to be fun on Android!

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ResourceType.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package io.moj.java.sdk.model.stream;
+
+/**
+ * Enumeration for activity stream resource types.
+ * Created by Mauro on 2016-06-24.
+ */
+public enum ResourceType {
+    VEHICLE("Vehicle"),
```
Comment (skidson) [inline mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ResourceType.java]: Same comment as above about @SerializedName

**Comment by skidson** [comment]
Couple minor comments, looks good!

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ActivityObjectType.java (hunk 1 above)
Comment (MauroHorie) [inline mojio-sdk-model/src/main/java/io/moj/java/sdk/model/stream/ActivityObjectType.java]: I would love to do that, but the problem is that there will be a Type field that would map to a different enum set depending on the subclass implementation. I could combine them all into a single enum but I'd rather not do that.

**Pull request merged**